data/tribes/initialization/empire/units.lua

```diff
@@ -1300,34 +1394,62 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_charcoal_kiln",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Charcoal Kiln
+            lore = pgettext("empire_building",
+               "‘Creta an carbone notandi?’<br>(Are they to be marked with chalk or charcoal?)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Charcoal Kiln
+            lore_author = pgettext("empire_building", "I.e. judge whether they are good or bad"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Charcoal Kiln
             purpose = pgettext("building", "Burns logs into charcoal.")
          }
       },
       {
          name = "empire_smelting_works",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Smelting Works
+            lore = pgettext("empire_building",
+               "‘Ignis aurum probat, miseria fortes viros.’<br>(Fire tests gold, adversity [tests] brave men.)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Smelting Works
+            lore_author = pgettext("empire_building", "Saledus on the importance of experienced soldiers"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Smelting Works
             purpose = pgettext("building", "Smelts iron ore into iron and gold ore into gold.")
          }
       },
       {
          name = "empire_toolsmithy",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Toolsmithy
+            lore = pgettext("empire_building",
+               "‘Faber est suæ quisque fortunæ.’<br>(Each man is the maker of his own fortune.)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Toolsmithy
+            lore_author = pgettext("empire_building", "But the toolsmith makes the tools for all of them"),
```

Comment:
I think continuing the lore in the lore author doesn't work nicely. So this should be adde dtomthe lore (even if getting lenghty and the author might be the motto  of the smithsguild or just stone tablet 4 of the economy handbook

Reply:
the problem is that then it doesn't match the original Latin… looks like I'll have to scrap this one

Reply:
well I think it would not harm to just continue the origin latin proverb. There is no need to stay to close to the roman lore here

Reply:
I used the advertising idea for author without changing the lore text. Do you think it works this way?

(I'd be uncomfortable inventing Latin phrases as the grammar is way too complicated)

Reply:
yes that might work